Tasks/AzureCLIV2/src/Utility.ts

```diff
@@ -39,16 +39,16 @@ export class Utility {
         let contents: string[] = [];
         contents.push(`$ErrorActionPreference = '${powerShellErrorActionPreference}'`);
         let filePath: string = tl.getPathInput("scriptPath", false, true);;
-        if (scriptLocation === "inlineScript") {
-            let inlineScript: string = tl.getInput("inlineScript", true);
-            filePath = path.join(tempDirectory, `azureclitaskscript${new Date().getTime()}_inlinescript.${fileExtensions[0]}`);
-            await Utility.createFile(filePath, inlineScript);
-        }
-        else{
+        if (scriptLocation.toLowerCase() === "scriptpath") {
```

Comment:
What is the scenario this change is trying to fix? scriptLocation is a required input with values either `scriptPath` or `inlineScript`. And for that the existing code should have been fine.

Reply:
+1, what scenario are we trying to fix here?

Reply:
As the validation for script Location value was missing. If user in YAML pipeline would have entered some wrong input, we would have defaulted to `inlineScript` in getScriptPath function. Thus by changing the order here, i was just ensuring that we always default to `inlineScript`.

But now i have added validation for scriptLocation to always be either of the two allowed values. Hence this change is no more required.